efficientdet/install_deps.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+for line in $(cat efficientdet/requirements.txt)
+do
+  pip install $line
+done
```

Comment:
Wouldn't the usual `pip install --requirement efficientdet/requirements.txt` be easier here?

Reply:
I want to install python packages in requirements.txt order.